src/main/java/com/mineinabyss/mobzy/registration/MobzyPacketInterception.kt

```diff
@@ -27,7 +25,8 @@ object MobzyPacketInterception {
             onSend(Server.SPAWN_ENTITY) {
                 PacketSpawnEntity(packet).apply{
                     entity(entityId).with<GearyEntityType>{
-                        packet.entityTypeModifier.write(0, EntityType.SNOWBALL) //TODO: Update to use ProtocolBurrito when it support entitytypes
+                        //TODO make a component to allow overriding the type here
+                        type = PacketEntityType.SNOWBALL.id
```

Comment:
Doesn't work, the SPAWN_ENTITY protocollib implementation expects an enum written as an entityTypeModifier.